src/components/Parser.js

```diff
@@ -29,6 +29,9 @@ function loading(file, callbackProgress, callbackEnd) {
   let start = 0;
   let end;
   let slice;
+  if (file === undefined) {
```

Comment:
Do you have any alert for this case?

Reply:
File become undefined when we click load button and then instead of choosing file we choose 'Close'.
So I don't think this case require any alerts and it'll even be distrubing

Reply:
Well, I've thought about case when we choose a file with incorrect format. It should be an alert, or maybe you can block selecting not json files. I think it should be in this task.

Reply:
Done. Check please

Reply:
Nice, thanks a lot!